test/passing/types.ml

```diff
@@ -50,3 +50,19 @@ val x :
     (* boooooom *)
     (** foooooooooooooooo foooooooooooooooooooooooo fooooooooooooooooooooooo
         fooooooooooooooooooo fooooooooooooooo*) ]
+
+type voting_period =
+    Tezos_client_alpha.Proto_alpha.Alpha_context.Voting_period.kind =
+  | Proposal
+  | Testing_vote
```

Comment:
What do others think about the indentation of line 55 in this example? It seems that both 2 and 4 spaces cause misleading vertical alignment. Would 6 be too much? I'm thinking that this situation is probably rare enough that 6 would be ok.

Reply:
I agree with your suggestion, I updated the code.